flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/consumer/SingleInputGateFactory.java

```diff
@@ -243,7 +244,10 @@ private InputChannel createKnownInputChannel(
 		return () -> bufferPoolFactory.createBufferPool(0, floatingNetworkBuffersPerGate);
 	}
 
-	private static class ChannelStatistics {
+	/**
+	 * Statistics of input channels.
```

Comment:
nit: Not really necessary because it does not provide any rich informations than the class name.

Reply:
The comment is only for style checking. Changing the class from package private to protected fails the style checking.